### PR TITLE
use module group name instead of src name as argument when `SendToGroup`

### PR DESCRIPTION
**Pull request by Congrool**
Signed-off-by: Congrool <chpzhangyifei@zju.edu.cn>

**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

For `beehiveContext.SendToGroup`, we'd better use common module group name. If using `model.SrcEdgeController` which indicates the `source` name, it's not easy to read.

Additionally, `model.SrcEdgeController` and `model.SrcDeviceController` seem nowhere to be used as `source` of message. So we can remove these redundant constants.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by vincentgoat — COMMENTED** [review]
lgtm

**Review by fisherxu — COMMENTED** [review]
/lgtm
/cc @zc2638

File: cloud/pkg/cloudhub/channelq/channelq.go [hunk 1]
```diff
@@ -316,9 +316,9 @@ func (q *ChannelMessageQueue) Publish(msg *beehiveModel.Message) error {
 	case application.MetaServerSource:
 		beehiveContext.Send(modules.DynamicControllerModuleName, *msg)
 	case model.ResTwin:
-		beehiveContext.SendToGroup(model.SrcDeviceController, *msg)
+		beehiveContext.SendToGroup(modules.DeviceControllerModuleGroup, *msg)
 	default:
-		beehiveContext.SendToGroup(model.SrcEdgeController, *msg)
+		beehiveContext.SendToGroup(modules.EdgeControllerGroupName, *msg)
```
Comment (zc2638) [inline cloud/pkg/cloudhub/channelq/channelq.go:321]: Is it possible to keep having the same naming format.
e.g.
`DeviceControllerModuleGroup` and `EdgeControllerModuleGroup`
e.g.
`DeviceControllerGroupName` and `EdgeControllerGroupName`

Comment (zc2638) [reply cloud/pkg/cloudhub/channelq/channelq.go:321]: Thanks a lot, it will be better if you have time to fix it.

**Pull request merged**